lib/browser/rpc.js

```diff
@@ -45,8 +44,17 @@ export function registerTypeConverter(type, handler) {
     typeConverters[type] = handler;
 }
 
-export function createSession() {
-    sessionId = sendRequest('create_session');
+export function createSession(host) {
+    let oldHost = sessionHost;
+
+    try {
+        sessionHost = host;
+        sessionId = sendRequest('create_session');
```

Comment:
if we pass `sessionHost` as a third argument to `send_request` to override the stored `sessionHost`. If we do this we don't need the try/catch here as we can set the stored `sessionHost` after the call.